modules/saml/src/Auth/Source/SP.php

```diff
@@ -160,13 +160,11 @@ public function getHostedMetadata(): array
         // add NameIDPolicy
         if ($this->metadata->hasValue('NameIDPolicy')) {
```

Comment:
If it has a value, it must be an array... We can use `getArray` instead of `getValue`, and we can drop the `is_array` test.